src/event-listeners/onRubberStampAnnotationAdded.js

```diff
@@ -0,0 +1,12 @@
+import core from 'core';
+import defaultTool from 'constants/defaultTool';
+import actions from 'actions';
+
+export default dispatch => rubberStampAnnotation => {
+  core.setToolMode(defaultTool);
+  core.getTool('AnnotationCreateRubberStamp').hidePreview();
+  dispatch(actions.closeElement('cursorOverlay'));
```

Comment:
This isn't needed now because the logic to show and hide the preview element is in WebViewer. if you merge master into this branch you should see that I deleted this in `onSignatureAnnotationAdded.js`